Source/Plugins/SyntheticBuilding/SyntheticBuildingFilters/InitializeSyntheticVolume.cpp

```diff
@@ -92,29 +88,8 @@ enum createdPathID : RenameDataPath::DataID_t
 //
 // -----------------------------------------------------------------------------
 InitializeSyntheticVolume::InitializeSyntheticVolume()
-: m_DataContainerName(SIMPL::Defaults::SyntheticVolumeDataContainerName)
-, m_CellAttributeMatrixName(SIMPL::Defaults::CellAttributeMatrixName)
-, m_EnsembleAttributeMatrixName(SIMPL::Defaults::CellEnsembleAttributeMatrixName)
-, m_LengthUnit(static_cast<int32_t>(IGeometry::LengthUnit::Micrometer))
-, m_InputStatsArrayPath(SIMPL::Defaults::StatsGenerator, SIMPL::Defaults::CellEnsembleAttributeMatrixName, SIMPL::EnsembleData::Statistics)
-, m_InputPhaseTypesArrayPath(SIMPL::Defaults::StatsGenerator, SIMPL::Defaults::CellEnsembleAttributeMatrixName, SIMPL::EnsembleData::PhaseTypes)
-, m_InputPhaseNamesArrayPath(SIMPL::Defaults::StatsGenerator, SIMPL::Defaults::CellEnsembleAttributeMatrixName, SIMPL::EnsembleData::PhaseName)
-, m_EstimateNumberOfFeatures(false)
-, m_EstimatedPrimaryFeatures("")
 {
-  m_Dimensions[0] = 128;
-  m_Dimensions[1] = 128;
-  m_Dimensions[2] = 128;
-
-  m_Spacing[0] = 0.25;
-  m_Spacing[1] = 0.25;
-  m_Spacing[2] = 0.25;
-
-  m_Origin[0] = 0.0f;
-  m_Origin[1] = 0.0f;
-  m_Origin[2] = 0.0f;
-
-  m_EstimatedPrimaryFeatures = "";
+  initialize();
```

Comment:
initialize() does nothing so we could get rid of it and make the constructor default.